yaspin/core.py

```diff
@@ -374,15 +378,11 @@ def fail(self, text: str = "FAIL") -> None:
         _text = text if text else "FAIL"
         self._freeze(_text)
 
+    def is_jupyter(self) -> bool:
```

Comment:
I am pretty sure that Jupyter is not the only front-end capable of intercepting the output stream.